src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Automation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+
+namespace Axe.Windows.AutomationTests
+{
+    [TestClass]
+    public class AutomationIntegrationTests
+    {
+        readonly string TestAppPath = Path.GetFullPath("../../../../tools/WildlifeManager/WildlifeManager.exe");
+        readonly string OutputDir = Path.GetFullPath("./TestOutput");
+
+        Process TestProcess;
+
+        [TestInitialize]
+        public void Startup()
+        {
+            LaunchTestApp();
+        }
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+            StopTestApp();
+
+            CleanupTestOutput();
+        }
+
+        [TestMethod]
+        [Timeout(10000)]
+        public void Scan_Integration()
+        {
+            var config = Config.Builder.ForProcessId(TestProcess.Id)
+                .WithOutputDirectory(OutputDir)
+                .WithOutputFileFormat(OutputFileFormat.A11yTest)
+                .Build();
+
+            var scanner = ScannerFactory.CreateScanner(config);
+
+            var output = scanner.Scan();
+
+            var regexForExpectedFile = $"{OutputDir.Replace("\\", "\\\\")}.*\\.a11ytest";
+
+            // Validate that we got the expected error count
+            Assert.AreEqual(12, output.ErrorCount);
+            Assert.AreEqual(12, output.Errors.Count());
```

Comment:
Can we remove the duplicated constant?

```suggestion
            Assert.AreEqual(output.ErrorCount, output.Errors.Count());
```

Reply:
Yes